spectator/logger/logger.go

```diff
@@ -1,8 +1,8 @@
 package logger
 
 import (
-	"log"
-	"os"
+	"fmt"
+	"log/slog"
```

Comment:
Eh, what is the reasoning behind this change?

Reply:
The default logger doesn't support levels, so all 3 levels were always shown when using the default logger. The only way to change that behavior was by implementing a custom logger. As a consequence we had an additional mechanism to control whether the payloads were logged or not (an env variable).

Instead, I changed to `slog` which supports levels out-of-the-box. Its default log level is Info (which sounds more reasonable to me). Now to log message payloads users just change the log level to default. 

So to sum up: simplicity and principle of least surprise

Reply:
My understanding is that the default Go logger is always like this, and it is incumbent upon the application owner to select a more capable logging package for their application, such as Uber/zap. There is enough variation in these selections, that I think we want to keep the most minimal interface here, to ensure that it works with any logger an app owner might choose.

Reply:
Oh, I see it's a new addition to the standard lib as of `1.21`:

> The new log/slog package in Go 1.21 brings structured logging to the standard library.

Excellent, good find.